modules/aws/kube-master/role.tf

```diff
@@ -147,3 +147,84 @@ resource "aws_iam_role_policy_attachment" "master_vpc_cni" {
   policy_arn = aws_iam_policy.master_vpc_cni[0].arn
   role       = aws_iam_role.master.name
 }
+
+
+data "aws_iam_policy_document" "master_ccm" {
```

Comment:
it's not necessary to create IAM Policy for ccm, just modify `data "aws_iam_policy_document" "master" {` is fine, please try below verson

```
data "aws_iam_policy_document" "master" {
  statement {
    sid = "EC2"

    actions = [
      "autoscaling:DescribeAutoScalingGroups",
      "autoscaling:DescribeLaunchConfigurations",
      "autoscaling:DescribeTags",
      "ec2:DescribeInstances",
      "ec2:DescribeRegions",
      "ec2:DescribeRouteTables",
      "ec2:DescribeSecurityGroups",
      "ec2:DescribeSubnets",
      "ec2:DescribeVolumes",
      "ec2:DescribeAvailabilityZones",
      "ec2:CreateSecurityGroup",
      "ec2:CreateTags",
      "ec2:CreateVolume",
      "ec2:ModifyInstanceAttribute",
      "ec2:ModifyVolume",
      "ec2:AttachVolume",
      "ec2:AuthorizeSecurityGroupIngress",
      "ec2:CreateRoute",
      "ec2:DeleteRoute",
      "ec2:DeleteSecurityGroup",
      "ec2:DeleteVolume",
      "ec2:DetachVolume",
      "ec2:RevokeSecurityGroupIngress",
      "ec2:DescribeVpcs",
    ]
    resources = [
      "*"
    ]
  }
```